modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -21,14 +21,16 @@ class AppointmentsController < VAOS::BaseController
       COMMENT = 'comment'
 
       def index
-        appointments[:data].each do |appt|
+        merged_appointments = appointments_service.merge_appointments(eps_appointments, appointments)
+
+        merged_appointments.each do |appt|
```

Comment:
`appointments_service.merge_appointments` is returning a hash of the form ` { data: [... appts array] }`. Is this supposed to iterate over the appts array?

Reply:
No longer does this (all controller logic removed)